src/yaml.jl

```diff
@@ -14,6 +14,11 @@ function dftramo_run(filename::AbstractString)
         (psi_previous, num_electrons_left, num_raMO) = import_checkpoint(ramoinput.checkpoint)
     else
         num_electrons_left = sum([get(E_DICT, n.atom.name, 0) for n in PeriodicAtomList(super)])
+        if !isequal(num_electrons_left/2, num_states(occ_states))
+            x = num_states(occ_states)
+            y = num_electrons_left/2
+            @warn "The number of occupied states $x does not match with the number of electrons $num_electrons_left ($y states) calculated. Consider adjusting your energy range."
```

Comment:
Just to be clear `@warn` only prints a warning message and does not stop execution. You'll want to change this to an `error()` call if you want to stop execution.

Reply:
There are some cases where we don't necessarily need the states and number of electrons to match, but we might consider inferring from the keys in the YAML or have a new key where it ignores this message and throws an error. I think a warning is sufficient for now.